src/collections/_documentation/development/server/environment.md

```diff
@@ -1,57 +1,38 @@
 
-### Linux
+#### A quick note on Bash vs Zsh...
 
-Follow [the Docker file in the repository](https://github.com/getsentry/sentry/blob/master/docker/Dockerfile) but instead of creating and installing a wheel, install the package in editable mode (`pip install . -e`).
+macOS Catalina will recommend that you use `zsh` instead of `bash`. While the differences are minimal, this guide primarily covers bash. You can verify your shell by typing `echo $SHELL`.
 
-### macOS
+## Clone the Repository
 
-{% capture __alert_content -%}
-macOS Catalina will recommend that you use the zsh shell, instead of bash. While the differences are minimal, this guide only covers bash, so for best results please use bash. You can verify this by typing `echo $SHELL`, which should result in `/bin/bash`.
-
-{%- endcapture -%}{%- include components/alert.html
-  title="Note"
-  content=__alert_content
-  level="warning"
-%}
-
-To get started, fork the repo at [https://github.com/getsentry/sentry](https://github.com/getsentry/sentry). Clone your forked repo and go into it:
+To get started, clone the repo at [https://github.com/getsentry/sentry](https://github.com/getsentry/sentry) - or, your fork if you are not a member of `@getsentry`.
 
 ```bash
-git clone https://github.com/<your github username>/sentry.git
+git clone https://github.com/getsentry/sentry.git
 cd sentry
 ```
 
-{% capture __alert_content -%}
-It's important that you remain in the repo's root directory for the rest of the setup! Also, please do the following steps in the exact order they're shown.
-{%- endcapture -%}{%- include components/alert.html
-  title="Note"
-  content=__alert_content
-  level="warning"
-%}
-
+You're going to be working out of the repository for the remainder of the setup.
 
-### macOS - System Dependencies
+## System Dependencies
 
 Install [Homebrew](http://brew.sh), and then run `brew bundle` to install the various system packages as listed in sentry's `Brewfile`.
 
 One thing that requires manual attention is `docker`, which should have just been installed. Open up Spotlight, search for "Docker" and start it. You should soon see the docker icon in your macOS toolbar. Docker will automatically run on system restarts, so this should be the only time you do this.
 
 You can verify that docker is running by running `docker ps`. If it doesn't error with something like `Error response from daemon: dial unix docker.raw.sock: connect: connection refused`, you're good to continue.
 
+## Python
 
-### macOS - Python
-
-Unfortunately, homebrew recently removed Python 2. While some versions of macOS ship with Python 2, it's recommended to not use the system's installation. Sentry also requires a specific version of Python, as shown in the file `.python-version`, and this may differ from the system's python.
+We utilize [pyenv](https://github.com/pyenv/pyenv) to install and manage python versions. It should have already been installed earlier when you ran `brew bundle`.
```

Comment:
If it wasn't installed earlier, manually install? Does this suggest that the install went sideways and the customer should check for other badness?

Reply:
yes but we dont need to spell it out, see above